### PR TITLE
[UPD] oca_dependencies

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
That's a comment for indicating why we put that repo. Why do you want to remove it?

**Comment by eilst** [comment]
Hey, I'm trying to fix the runbot build problem. It fails and I found that after this commit https://github.com/OCA/server-auth/commit/dbca5bfcb7dd252ebbcb239981a31a41e41f7b59 it started. I thought this might have been the problem.

**Comment by pedrobaeza** [comment]
I don't think that's the problem...

Put an issue with the problem you face and then people can give you a tip.

**Comment by eilst** [comment]
OK, I will do that and also try to figure it out. Thanks

**Comment by eilst** [comment]
It seems the problem is in server-backend/requirements.txt'

  subprocess.CalledProcessError: Command '['pip', 'install', '--no-binary', 'pycparser', '-Ur', '/home/odoo/dependencies/server-backend/requirements.txt']' returned non-zero exit status 1

File: requirements.txt [hunk 1]
```diff
@@ -8,4 +8,4 @@ pyotp
 # auth_u2f
 python-u2flib-server
 responses
-
+libmysqlclient-dev
```
Comment (eilst) [inline requirements.txt]:
```suggestion
```

**Comment by eilst** [comment]
Solution: https://github.com/OCA/server-auth/pull/123

**Pull request closed without merging**